react_app/src/App.scss

```diff
@@ -0,0 +1,183 @@
+.App {
+  font-family: sans-serif;
+  text-align: center;
+}
+
+html {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  -webkit-box-sizing: inherit;
+          box-sizing: inherit;
+}
+
+body {
+  outline : none;
+  margin: 20px 0;
+  padding: 0;
+  line-height: 1;
+  font-size: 16px;
+  font-family: 'Roboto', sans-serif;
+  color: #202020;
+  background-color: #fbfbfb;
+  font-smooth: always;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.container {
+  max-width: 1140px;
+  width: 100%;
+  margin: auto;
+}
+
+@mixin styling() {
+  background-color: white;
+  border-color: #ccc;
+  border-radius: 4px;
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: 0 .125rem .25rem rgba(0,0,0,.075) !important;
```

Comment:
spaces after commas. Also `!important` is _eeeevil_! If you *have* to use it to override something that you can't find a workaround for, plz add justification via comment.

Reply:
oops removed!